european-journal-of-paediatric-neurology.csl

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" page-range-format="minimal" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>European Journal of Paediatric Neurology</title>
+    <title-short>EJPN</title-short>
+    <id>http://www.zotero.org/styles/european-journal-of-paediatric-neurology</id>
+    <link href="http://www.zotero.org/styles/european-journal-of-paediatric-neurology" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-vancouver" rel="template"/>
+    <link href="http://www.elsevier.com/journals/european-journal-of-paediatric-neurology/1090-3798/guide-for-authors" rel="documentation"/>
+    <author>
+      <name>Sankar</name>
+      <email>s.murugesan@elsevier.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>1090-3798</issn>
+	<issn>1532-2130</issn>
```

Comment:
@Sankar-GEO-ELS, for future submissions, could you use the `<eissn/>` element for the electronic ISSN? (in this case, 1532-2130 is the eISSN, see http://www.ncbi.nlm.nih.gov/nlmcatalog?term=1090-3798%5BISSN%5D )

Thanks!

Changed in https://github.com/citation-style-language/styles/commit/e0918a75a0b7840282cdefc764d0b6b42c024842